docs/index.asciidoc

```diff
@@ -207,8 +207,31 @@ Stop entire pipeline when encountering a non decodable message.
 
 Convert data types to match the protobuf definition (if possible).
 The protobuf encoder library is very strict with regards to data types. Example: an event has an integer field but the protobuf definition expects a float. This would lead to an exception and the event would be lost.
+
 This feature tries to convert the datatypes to the expectations of the protobuf definitions, without modifying the data whatsoever. Examples of conversions it might attempt:
-"true" string => true boolean
-17 int => 17.0 float
-12345 number => "12345" string
+
+  `"true" :: string => true :: boolean`
+
+  `17 :: int => 17.0 :: float`
+
+  `12345 :: number => "12345" :: string`
+
+Available only for protobuf version 3.
+
+[id="plugins-{type}s-{plugin}-pb3_set_oneof_metainfo"]
+===== `pb3_set_oneof_metainfo`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is false
+
+Add meta information to `[@metadata][pb_oneof]` about which classes were chosen for [oneof](https://developers.google.com/protocol-buffers/docs/proto3#oneof) fields.
```

Comment:
nit: On the documentation perhaps we should make clear that the end value of the `[@metadata][pb_oneof]` field would look like:

```ruby
[@metadata][pb_oneof][field_name]
```

nit: Also perhaps that the field would be populated recursively for the nested fields as shown in the `pb3decoder_test8c` test.